elytron/src/main/java/org/wildfly/extension/elytron/AuditResourceDefinitions.java

```diff
@@ -106,6 +107,12 @@ class AuditResourceDefinitions {
             .setRestartAllServices()
             .build();
 
+    static final SimpleAttributeDefinition ENCODING = new SimpleAttributeDefinitionBuilder(ElytronDescriptionConstants.ENCODING, ModelType.STRING, true)
+            .setAllowExpression(true)
```

Comment:
As the new attribute allow expressions it should be tested with an expression set.